### PR TITLE
[Do not merge] spring deps to fix springboot module on openshift

### PR DESCRIPTION
**Pull request by desmax74**
@mariofusco @jiripetrlik @danielezonca

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,8 +27,8 @@
     <version.logback.kafka.appender>0.2.0-RC2</version.logback.kafka.appender>
     <version.servlet>4.0.1</version.servlet>
     <version.slf4j>1.7.26</version.slf4j>
-    <version.org.springframework>5.1.3.RELEASE</version.org.springframework>
-    <version.org.spring-boot>2.1.1.RELEASE</version.org.spring-boot>
+    <version.org.springframework>5.2.0.M3</version.org.springframework>
```
Comment (jiripetrlik) [inline pom.xml]: It is only pre-release build.

Comment (mareknovotny) [reply pom.xml]: Hey @desmax74 we should use GAV for dependencies which are available in Maven Central or in jboss.org repository
See our own rules https://github.com/kiegroup/droolsjbpm-build-bootstrap#requirements-for-dependencies

Comment (desmax74) [reply pom.xml]: @mareknovotny @jiripetrlik  this issue isn't for merge, is to track the changes needed to fix the LeaderElection from Camel when the Springboot is used, adding in the description

Comment (desmax74) [reply pom.xml]: @mareknovotny  @jiripetrlik  Fixed by https://github.com/kiegroup/openshift-drools-hacep/pull/17

**Pull request closed without merging**